streaming/src/main/scala/org/apache/gearpump/streaming/appmaster/ClockService.scala

```diff
@@ -261,7 +261,8 @@ class ClockService(
     case GetCheckpointClock =>
       sender ! CheckpointClock(minCheckpointClock)
 
-    case GetStallingTasks =>
+    // Please do not remove the `getStalling`
+    case getStalling: GetStallingTasks =>
```

Comment:
If the field of `GetStallingTasks` is never used, we'd better turn it into a case object